test/second_q/mappers/test_ternary_tree_mapper.py

```diff
@@ -0,0 +1,185 @@
+# This code is part of a Qiskit project.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test Ternary Tree Mapper """
+
+import unittest
+from test import QiskitNatureTestCase
+
+import numpy as np
+from qiskit.circuit import Parameter
+from qiskit.quantum_info import SparsePauliOp
+from qiskit.algorithms.eigensolvers import NumPyEigensolver
```

Comment:
```suggestion
from qiskit_algorithms import NumPyEigensolver
```
qiskit.algorithms has been deprecated for a while as the code was moved out of Qiskit into a separate repo. While you can import it direct from the subfolder, all the algorithms are importable from the top level, so I suggest to change it to that. It failed across the board when I updated the branch and re-ran this as things are now tested against the latest Qiskit i.e. 1,0 in which qiskit.algorithms no longer exists having finally been removed following the earlier deprecation.

Reply:
Thanks, I missed this.